composer.json

```diff
@@ -44,18 +43,16 @@
         "composer/bin/test-reporter"
     ],
     "scripts": {
-        "post-install-cmd": "Tooly\\ScriptHandler::installPharTools",
-        "post-update-cmd": "Tooly\\ScriptHandler::installPharTools"
+        "box": [
+            "composer install --no-dev --prefer-dist",
```

Comment:
Installing non-development dependencies here to make sure that

* dependencies are installed
* only interesting dependencies are installed